command/remote.go

```diff
@@ -57,10 +60,13 @@ func (c *RemoteCommand) Run(args []string) int {
 
 	// Populate the various configurations
 	c.remoteConf.Config = map[string]string{
-		"address":      address,
-		"access_token": accessToken,
-		"name":         name,
-		"path":         path,
+		"address":        address,
+		"access_token":   accessToken,
+		"security_token": securityToken,
+		"name":           name,
+		"path":           path,
+		"bucket":         bucket,
```

Comment:
I would just make bucket part of path. e.g. the path should be specified as s3://bucket/prefix/